src/ogkush.js

```diff
@@ -4744,17 +4744,17 @@ class OGInfinity {
     sum &&
       this.json.empire.forEach((planet) => {
         mlvl += Number(planet[1]);
-        mprodh += Number(planet.production.hourly[0]);
-        mprodd += Number(planet.production.daily[0]);
-        mprodw += Number(planet.production.weekly[0]);
+        mprodh += isNaN(Number(planet.production.hourly[0]))? 0 : Number(planet.production.hourly[0]);
```

Comment:
You can set like that, i think.

```suggestion
        mprodh +=Number(planet.production.hourly?[0] || 0);
```

Reply:
It seems a nice and compact way to use a default do the sanitization, what do you think @blag001 ?

Reply:
done ;)